idpi/test/fe_templates/test_destagger.nl

```diff
@@ -0,0 +1,130 @@
+&RunSpecification
+ additional_profiling=.true.
+ strict_nl_parsing=.true.
+ print_product_list=.true.
+ product_list_keep_epsm=0,1,10
+ product_list_keep_leadtime_hh=0,1,2,3,6,9,12,18,24,33,45
+ verbosity="moderate"
+ diagnostic_length=110
+ soft_memory_limit=350.0 
+ wait_time=10
+ mx_wait_time=14400
+ enable_repeat_mode='all'
+ ready_flag_dir="{{ready_flags}}"
+   ready_flag_prefix_pure="FXTRF_PURE_"
+   ready_flag_prefix_other="FXTRF_OTHER_"
+ out_noready_postfix=".incomplete"
+ out_snapshot_postfix=".snapshot_"
+ stop_flag="/opr/osm/rh7.9/tmp/21101512_407/.ppStopTasks_c"
+ n_ompthread_total=1
+ n_ompthread_collect=1
+ n_ompthread_generate=1,1
+ out_cost_expensive=1000
+/
+&GlobalResource
+ dictionary="/oprusers/osm/opr/config/resources/dictionary_cosmo.txt"
+ grib_definition_path="/oprusers/osm/opr/config/resources/eccodes_definitions_cosmo", 
```

Comment:
The paths to /oprusers/osm/opr must be replaced by paths to /project/s83c/fieldextra/tsa/, since we work with /project/s83c/fieldextra/tsa/bin/fieldextra_gnu_opt_omp (latest stable version of develop branch that has passed all tests except for the production test).

Reply:
great, thanks I changed it